test/integration/discovery-consul.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+load helpers
+
+# Address on which Consul will listen (random port between 8000 and 9000).
+CONSUL_HOST=127.0.0.1:$(( ( RANDOM % 1000 )  + 8000 ))
+
+# Container name for integration test
+CONTAINER_NAME=swarm_consul
+
+function check_leader() {
+	docker_host logs $CONTAINER_NAME | grep "New leader elected"
+}
+
+function start_consul() {
+	run docker_host run -h $CONTAINER_NAME -p $CONSUL_HOST:8500 -d progrium/consul -server -bootstrap-expect 1 -data-dir /test
```

Comment:
The documentation specifes to use of -bootstrap-expect. -bootstrap was used in prior versions and for manual setup. https://www.consul.io/docs/guides/bootstrapping.html.